imsim/opsim_data.py

```diff
@@ -365,6 +374,10 @@ def OpsimData(config, base, value_type):
     """
     meta = galsim.config.GetInputObj('opsim_data', config, base, 'OpsimData')
 
+    if 'disable_iers_downloads' not in galsim.config.eval_base_variables:
+        galsim.config.eval_base_variables.append('disable_iers_downloads')
+        base['disable_iers_downloads'] = meta.disable_iers_downloads
```

Comment:
I don't think this is the way we want to do this.  You're effectively making our default template (which uses "$disable_iers_downloads") implicitly dependent on OpsimData.  If a user doesn't include that anywhere, then the default template file will be broken in a very hard to diagnose way.

You should either have the template explicitly include this in eval_variables and have both batoid_wcs and opsim_data use that as the source information, or have one of them use a "Current" item to get it from the other one, so at least the user can see the way they are dependent on each other.  E.g. 
```
wcs:
    type: Batoid
    disable_iers_downloads: "@input.opsim_data.disable_iers_downloads"
```
If the latter, then you should add a note that if the user isn't using either of our two forms of opsim_data (skycat or instcat) then this should be set by hand in the wcs section.

Reply:
Also, do we really need it in both places?  It seems like the implementation is such that if either one has it, then it will get implemented and the astropy global variables will be set correctly.  So maybe the better solution is just to identify which one needs it to be set correctly first.  (I would expect opsim_data, but I might be wrong.)  Then just do it there and don't bother with having it also done in the other place.

I'll also reiterate my suggestion to make the default True.  I don't think we probably have any use cases where we want to guarantee that these are fully up to date, so letting it always be disabled seems like the better option to me.

Reply:
The problem is that each process needs to have the download disabled, so if `output.nproc > 1`, the distributed processes, i.e., the ones running batoid for each CCD, would still try to do the download even if it's already been disabled for opsim_data in the head process.

Reply:
OK.  Is there a need to allow `disable_iers_downloads=False`?  Why not just hard-code this entirely and always make it disabled when using imsim?

Reply:
Making `disable_iers_downloads=False` the default seems fine, and I don't personally have a problem with hard-coding it here, but maybe someone would want it enabled in some cases? I really couldn't say.  @jmeyers314 @cwwalter Do you have an opinion?

Reply:
I can't imagine why I'd ever care about the precise iers numbers.  This is basically degenerate with a pointing error when aiming the telescope, but probably much smaller.